docs/reference_keymap_extras.md

```diff
@@ -20,6 +20,7 @@ These headers are located in [`quantum/keymap_extras/`](https://github.com/qmk/q
 
 |Layout                           |Keycodes Header                  |Sendstring LUT Header               |
 |---------------------------------|---------------------------------|------------------------------------|
+|Canadian French                  |`keymap_canadian_french.h`       |`sendstring_canadian_french.h`      |
```

Comment:
I think I would name this `French (Canada)` to match the other entries where the convention is `Language (Region)`. Then it should be moved down beneath the Belgian French entry.

Reply:
Applied in 12815db88f.